src/config/validate/runtime.ts

```diff
@@ -67,7 +91,7 @@ export const schema = Joi.object({
     baker_pays_transaction_fee: Joi.boolean(),
     minimum_amount: Joi.number().positive(),
   },
-  notifications: Joi.array(),
+  notifications: Joi.array().items(validPlugin),
```

Comment:
Technically anything with valid `id` and `loadNotificationPlugin` should be `validPlugin`